subxt/src/blocks/extrinsic_types.rs

```diff
@@ -217,12 +232,10 @@ where
         // Skip over the first byte which denotes the version and signing.
         let cursor = &mut &bytes[1..];
 
-        let mut address_start_idx = 0;
-        let mut address_end_idx = 0;
-
-        if is_signed {
-            address_start_idx = bytes.len() - cursor.len();
-
+        let signed = if !is_signed {
```

Comment:
Nit: maybe call this `signed_details` for clarity.

Reply:
We could do that, but for the `?` operator used for error handling inside the block this would mean, we have 
```rs
let signed_details = is_signed
            .then(|| -> Result<SignedExtrinsicDetails, Error> {
                // use of `?` to handle decoding errors...
                ..
                Ok(SignedExtrinsicDetails {..}) 
            }).transpose()?;
```
I personally find it a bit more confusing.

Reply:
Good point; let's leave it as is then :)